### ISSUE TITLE
GCP - External resource attribute not supported in TF module "terraform-google-modules/vpc-service-controls/google//modules/regular_service_perimeter"

### ISSUE THREAD

**Issue by nturner-maritz** [terraform-google-modules/terraform-google-vpc-service-controls#143]
### TL;DR

Hi,

We have a requirement to transfer data between S3 (AWS) and BigQuery (GCP) using Amazon S3 BigLake tables.

As the project is protected by VPC SC Perimeter with BigQuery as one of the restricted service within the perimeter, we are trying to create an egress rule on the perimeter and trying to specify "external_resources" but this is not supported in the terraform module "terraform-google-modules/vpc-service-controls/google//modules/regular_service_perimeter", is there any plan to enhance this module to accept "external_resources" in ingress/egress rules?

Note that currently it's supported in Terraform resource "google_access_context_manager_service_perimeter"

Thanks,
Nicholas.

### Terraform Resources

```markdown
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/access_context_manager_service_perimeter.html#external_resources
```

### Additional information

Clone of issue #104

**Comment by nturner-maritz** [comment]
Clone of issue https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/issues/104
